holoviews/core/sheetcoords.py

```diff
@@ -515,13 +515,18 @@ def _boundsspec2slicespec(boundsspec,scs):
         t_m,l_m = scs.sheet2matrix(l,t)
         b_m,r_m = scs.sheet2matrix(r,b)
 
-        l_idx = int(np.ceil(l_m-0.5))
-        t_idx = int(np.ceil(t_m-0.5))
+        l_idx = np.ceil(l_m-0.5)
+        t_idx = np.ceil(t_m-0.5)
         # CBENHANCEMENT: Python 2.6's math.trunc()?
-        r_idx = int(np.floor(r_m+0.5))
-        b_idx = int(np.floor(b_m+0.5))
+        r_idx = np.floor(r_m+0.5)
+        b_idx = np.floor(b_m+0.5)
 
-        return t_idx,b_idx,l_idx,r_idx
+        # sometimes contains NaNs so just set them to 0
+        # rather than raising an exception
```

Comment:
I'm not completely sure if this is the right thing to do. If our lbrt coordinates are nans, isn't that usually a sign something has gone wrong beforehand?

Do you still see this with the changes in `holoviews/core/data/xarray.py`?

Reply:
You're right; it seems I don't have this issue anymore. I'll remove this then.